proposed/kubeflow/v0.6.2/questions.yml

```diff
@@ -0,0 +1,158 @@
+categories:
+- machine learning
+rancher_max_version: 2.3.3
+labels:
+  io.rancher.certified: experimental
+namespace: kubeflow
+questions:
+- variable: profiles.admin
+  default: johnDoe@acme.com
```

Comment:
is the admin required, if not would prefer to set the default value an empty string.

Reply:
Yes, it's required. Updated setting required true.